python/ray/worker.py

```diff
@@ -673,7 +686,7 @@ def export_remote_function(self, function_id, function_name, function,
             })
         self.redis_client.rpush("Exports", key)
 
-    def run_function_on_all_workers(self, function):
+    def run_function_on_all_workers(self, function, include_driver=False):
```

Comment:
Document this argument. Also, let's rename the argument to something more understandable. How about `run_on_other_drivers`?

Reply:
Sure.